src/ray/pubsub/publisher.h

```diff
@@ -133,7 +144,7 @@ class Subscriber {
   /// It becomes a nullptr whenever the long polling request is replied.
   std::unique_ptr<LongPollConnection> long_polling_connection_;
   /// Queued messages to publish.
-  std::list<std::unique_ptr<rpc::PubsubLongPollingReply>> mailbox_;
+  std::queue<std::unique_ptr<rpc::PubsubLongPollingReply>> mailbox_;
```

Comment:
nit: will this sometimes not efficient as list? for example, if resize happened a lot?

Reply:
This is very minor but `std::queue` uses `std::deque` underneath which does not copy on resize.